view/theme/frio/style.php

```diff
@@ -225,7 +225,7 @@
 header('Last-Modified: ' . $modified);
 
 // Only send the CSS file if it was changed.
-if (isset($_SERVER['HTTP_IF_MODIFIED_SINCE']) || isset($_SERVER['HTTP_IF_NONE_MATCH'])) {
+if (isset($_SERVER['HTTP_IF_MODIFIED_SINCE']) && isset($_SERVER['HTTP_IF_NONE_MATCH'])) {
```

Comment:
Shouldn't it be a `defaults()` call instead? I'm not sure both headers are required to potentially send a 304.

Reply:
I think that both are needed.

Reply:
According to MDN, both are mutually exclusive: 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-None-Match

Reply:
Possibly the following coding after the "if" is wrong, since it compares **both** values and will only return a "not modified" if **both** values are identically.

I can add a note at this, so that we check if this works at all. It wouldn't like to leave this as a blocker.

Reply:
The behaviour in mod/photo.php is completely different. I guess we have to rework the whole part. But this should be done in a different PR.